### PR TITLE
practice problem 00

### PR DESCRIPTION
**Pull request by KerryKDiehl**
"Hello world"

### CODE REVIEW

**Comment by KerryKDiehl** [comment]
@sophylee do you want me to put descriptions in each of these pull requests?

File: lib/00_hello.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def hello
+  "Hello!"
```
Comment (sophylee) [inline lib/00_hello.rb:2]: Should be a `puts ` before the string.

File: lib/00_hello.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+end
+
+def greet(name)
+  "Hello, #{name}!"
```
Comment (sophylee) [inline lib/00_hello.rb:6]: A `puts ` here too.

**Review by sophylee — APPROVED** [review]
Works, but I added a suggestion.

**Comment by sophylee** [comment]
> @sophylee do you want me to put descriptions in each of these pull requests?

For these I usually know what the exercise is about so a simple comment is ok if it's not apparent from the PR title what the PR is about.

File: lib/00_hello.rb (hunk 1 above)
Comment (KerryKDiehl) [reply lib/00_hello.rb:2]: Based on the tests that were provided, this is actually just looking for a return value, not for the phrases to be actually printed out.  Seems counter-intuitive to every "Hello World" exercise I've ever run into but perhaps they're just trying to keep me on my feet :)

**Pull request closed without merging**